src/deadline/maya_adaptor/MayaClient/render_handlers/default_maya_handler.py

```diff
@@ -94,9 +97,31 @@ def start_render(self, data: dict) -> None:
         maya.cmds.setAttr("defaultRenderGlobals.endFrame", frame)
 
         camera = self.get_camera_to_render(data)
-
         print(f"Rendering camera: {camera}", flush=True)
 
+        # In order of preference, use the task's output_file_prefix, the step's output_file_prefix, or the scene file setting.
+        output_file_prefix = data.get("output_file_prefix", self.output_file_prefix)
+        if output_file_prefix:
+            maya.cmds.setAttr(
+                "defaultRenderGlobals.imageFilePrefix", output_file_prefix, type="string"
+            )
+
+        if self.image_width is not None:
+            maya.cmds.setAttr("defaultResolution.width", self.image_width)
+            print(f"Set image width to {self.image_width}", flush=True)
+        if self.image_height is not None:
+            maya.cmds.setAttr("defaultResolution.height", self.image_height)
+            print(f"Set image height to {self.image_height}", flush=True)
+
+        region = [
```

Comment:
Q: Why do we retrieve and validate the region values, if we don't use them afterwards.

Reply:
Because if the job tries to do a region render when the render handle doesn't support it, the render should fail instead of producing incorrect output.